python/cugraph/structure/graph.py

```diff
@@ -177,17 +177,17 @@ def delete_edge_list(self):
 
     def add_adj_list(self, offset_col, index_col, value_col=None, copy=False):
         """
-        Create the adjacency list representation of a Graph. The passed
-        offset_col and index_col arguments wrap gdf_column objects that
-        represent a graph using the adjacency list format. If value_col is
-        None, an unweighted graph is created. If value_col is not None, an
-        weighted graph is created. If copy is False, this function stores
-        references to the passed objects pointed by offset_col and index_col.
-        If copy is True, this funcion stores references to the deep-copies of
-        the passed objects pointed by offset_col and index_col. If this class
-        instance already stores a graph, invoking this function raises an
-        error. Undirected edges must be stored as directed edges in both
-        directions.
+        Initialize a graph from the adjacency list. It is error to call this
```

Comment:
'It is error' -> 'It is erroneous'.

Reply:
Thanks for finding these typos, I made a fix.